src/Driver/driver_configs.jl

```diff
@@ -131,14 +130,19 @@ function AtmosLESConfiguration(
     numerical_flux_first_order = RusanovNumericalFlux(),
     numerical_flux_second_order = CentralNumericalFluxSecondOrder(),
     numerical_flux_gradient = CentralNumericalFluxGradient(),
+    grid_stretching = SVector{3}(
+        NoStretching(),
+        NoStretching(),
+        NoStretching(),
+    ),
```

Comment:
Suggest moving this just above or below the `meshwarp` argument, to group related parameters together. Also:

```suggestion
    grid_stretching = [
        NoStretching(),
        NoStretching(),
        NoStretching(),
    ],
```